plugins/kafka/librdkafka

```diff
@@ -0,0 +1 @@
+Subproject commit 3e1babf4f26a7d12bbd272c1cdf4aa6a44000d4a
```

Comment:
This sha is the current head of the master branch of librdkafka - we should probably be targeting a stable branch/tag ?